src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+msgid "Click here to open tag menu for {tag}"
+msgstr "Klik hier om het tagmenu voor {tag} te openen"
+
+#: src/components/dialogs/PostInteractionSettingsDialog.tsx:304
+msgid "Click to disable quote posts of this post."
+msgstr "Klik om citaatberichten voor dit bericht uit te schakelen."
+
+#: src/components/dialogs/PostInteractionSettingsDialog.tsx:305
+msgid "Click to enable quote posts of this post."
+msgstr "Klik om citaatberichten voor dit bericht in te schakelen."
+
+#: src/components/dms/MessageItem.tsx:240
+msgid "Click to retry failed message"
+msgstr "Klik om mislukt privébericht opnieuw te proberen"
+
+#: src/screens/Onboarding/index.tsx:32
+msgid "Climate"
+msgstr "Klimaat"
+
+#: src/components/dms/ChatEmptyPill.tsx:39
+msgid "Clip 🐴 clop 🐴"
+msgstr "Clip 🐴 clop 🐴"
+
+#: src/components/dialogs/GifSelect.tsx:282
+#: src/components/dialogs/VerifyEmailDialog.tsx:246
+#: src/components/dms/dialogs/SearchablePeopleList.tsx:263
+#: src/components/NewskieDialog.tsx:146
+#: src/components/NewskieDialog.tsx:153
+#: src/components/StarterPack/Wizard/WizardEditListDialog.tsx:117
+#: src/components/StarterPack/Wizard/WizardEditListDialog.tsx:123
+#: src/view/com/modals/ChangePassword.tsx:268
+#: src/view/com/modals/ChangePassword.tsx:271
+#: src/view/com/util/post-embeds/GifEmbed.tsx:197
+msgid "Close"
+msgstr "Sluiten"
+
+#: src/components/dialogs/GifSelect.tsx:169
+msgid "Close GIF dialog"
+msgstr "Sluit GIF-dialoogvenster"
+
+#: src/components/Dialog/index.web.tsx:110
+#: src/components/Dialog/index.web.tsx:256
+msgid "Close active dialog"
+msgstr "Sluit actief dialoogvenster"
+
+#: src/screens/Login/PasswordUpdatedForm.tsx:32
+msgid "Close alert"
+msgstr "Sluit waarschuwing"
+
+#: src/view/com/util/BottomSheetCustomBackdrop.tsx:36
+msgid "Close bottom drawer"
+msgstr "Sluit onderste lade"
+
+#: src/components/dialogs/GifSelect.tsx:276
+msgid "Close dialog"
+msgstr "Sluit dialoogvenster"
+
+#: src/view/com/lightbox/ImageViewing/components/ImageDefaultHeader.tsx:36
+msgid "Close image"
+msgstr "Sluit afbeelding"
+
+#: src/view/com/lightbox/Lightbox.web.tsx:129
+msgid "Close image viewer"
+msgstr "Sluit afbeeldingsviewer"
+
+#: src/view/shell/index.web.tsx:67
+msgid "Close navigation footer"
+msgstr "Sluit navigatievoettekst"
+
+#: src/components/Menu/index.tsx:229
+#: src/components/TagMenu/index.tsx:261
+msgid "Close this dialog"
+msgstr "Sluit dit dialoogvenster"
+
+#: src/view/shell/index.web.tsx:68
+msgid "Closes bottom navigation bar"
+msgstr "Sluit onderste navigatiebalk"
+
+#: src/screens/Login/PasswordUpdatedForm.tsx:33
+msgid "Closes password update alert"
+msgstr "Sluit waarschuwing voor bijwerken wachtwoord"
+
+#: src/view/com/composer/Composer.tsx:552
+#: src/view/com/lightbox/ImageViewing/components/ImageDefaultHeader.tsx:37
+msgid "Closes viewer for header image"
+msgstr "Sluit viewer voor headerafbeelding"
+
+#: src/view/com/notifications/FeedItem.tsx:265
+msgid "Collapse list of users"
+msgstr "Dichtvouwen gebruikerslijst"
```

Comment:
Are you certain you don't want "inklappen" and "uitklappen" (collapse expand). Dichtvouwen and openvouwen sounds very tactile.

Reply:
My MSc thesis was on tactile feedback, and that still shows :-) Looking at frequency of term use: changed to inklappen and uitklappen.